mkosi/tree.py

```diff
@@ -79,6 +79,10 @@ def copy_tree(config: MkosiConfig, src: Path, dst: Path, *, preserve_owner: bool
         run(copy)
 
 
+def rmtree(path: Path) -> None:
+    run(["rm", "-rf", path])
```

Comment:
It's rarely an issue and shouldn't be one here in most cases, but let's make sure that `rm` doesn't parse a path as an option. 
```suggestion
    run(["rm", "-rf", "--", path])
```

Maybe something we should do in other places as well. 

The function does give me a bit of pause. I really hope we never nuke somebody's home by accident.